Assignment-1/main.swift

```diff
@@ -0,0 +1,179 @@
+
+import Foundation
+
+class ItemDetails{
+     var name: String
+     var price: Double
+    var quantity: Int? = 1
```

Comment:
Ident code,  Select the part of code and press ctrl + I ( Shortcut for identing)